src/App.js

```diff
@@ -10,29 +10,35 @@ class App extends Component {
     super();
     this.state = {
       movies:[],
-       error:null
+
+      error:null,
+      selected: null
+
     }
     
   }
 
   showMore = (id) => {
```

Comment:
Wondering if you might be able to come up with a more descriptive name for this function?